__tests__/app.test.js

```diff
@@ -160,6 +161,25 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200 the number of articles associated with given topic has the correct length", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        expect(article.length === 12).toBe(true);
```

Comment:
I think testing for the length is ok however I would like you to loop through the returned value and check that each topic is in fact what you expect it to be.